tlatools/org.lamport.tlatools/src/tla2sany/semantic/SymbolTable.java

```diff
@@ -2,7 +2,9 @@
 // Portions Copyright (c) 2003 Microsoft Corporation.  All rights reserved.
 package tla2sany.semantic;
 
-import tla2sany.utilities.Stack;
+import java.util.ArrayDeque;
```

Comment:
Why is this change necessary? The collections in `java.util` and those in `tla2sany.util` are unlikely to be fully compatible.

Reply:
If this is absolutely necessary and cannot be avoided, isolate the code into a dedicated commit.

Reply:
I will cherry-pick this commit out into another branch for submission in the future. Primarily I made it because the state of our custom-written `Stack` class is hard to understand when viewed in the debugger. It would be nice to get rid of it eventually.